### DISCUSSION TITLE
device-registration.ts 실행시 오류

### DISCUSSION THREAD

**Discussion by disabled-variable** [storycraft/node-kakao#678] [Q&A] [answered]
examples의 device-registration.ts 실행시 아래와 같은 오류가 나옵니다.
해결책이 무엇인가요?

현재 저의 코드입니다.
![image](https://user-images.githubusercontent.com/84914862/134761075-44952b33-c23e-474b-a4f7-e5d1438cb596.png)
![image](https://user-images.githubusercontent.com/84914862/134761089-64a4be8f-0269-4c51-bef8-2b61454f93b8.png)

실행결과입니다.
![image](https://user-images.githubusercontent.com/84914862/134761022-c92dc733-bfd8-460f-bcbf-687970e44be4.png)

**Comment by kiwioverflow** [comment]
https://github.com/storycraft/node-kakao/blob/967d0355f45b108a282d94d20547e37b2670b161/src/api/auth-api-client.ts#L104

같이 오는 메세지도 확인해보시길 바랍니다.

**Reply by disabled-variable** [reply]
모든 정보를 제대로 기입했는데 에러가 나와서 질문드렸습니다..

**Comment by generalxd** [comment, marked as answer]
process.env[] 를 그냥 문자열로바꿔보셔도 안되나요 ? 글구 비번은 가리시지 ㅎㅎ..

**Reply by disabled-variable** [reply]
문자열로 바꾸니까 다른 오류코드로 바뀌었어요!!
비밀번호는 가계정이라 괜찮습니다 감사합니다!
